src/psyclone/tests/domain/gocean/transformations/gocean_opencl_trans_test.py

```diff
@@ -532,6 +538,9 @@ def test_opencl_routines_initialisation(kernel_outputdir):
         field%device_ptr = transfer(create_rw_buffer(size_in_bytes), \
 field%device_ptr)
         field%data_on_device = .true.
+        ! psyclone codeblock reason:
+        !  - unsupported statement
```

Comment:
I'm wondering whether we should filter-out duplicates when constructing this message? I guess that would probably make it harder to relate them to specific statements.

Reply:
There will be less duplicates now that we included the type(fp2node), but even when there are I prefer them (unless we end up with the end comment)

Reply:
OK.